services/graph/pkg/service/v0/option.go

```diff
@@ -42,6 +43,7 @@ type Options struct {
 	SearchService            searchsvc.SearchProviderService
 	KeycloakClient           keycloak.Client
 	EventHistoryClient       ehsvc.EventHistoryService
+	Store                    microstore.Store
```

Comment:
What is this good for? I might be overlooking something, but is it actually used anywhere?